apps/SageAccountWeb/package.json

```diff
@@ -12,7 +12,7 @@
     "@mui/system": "^5.11.16",
     "@mui/utils": "^5.11.13",
     "lodash-es": "^4.17.21",
-    "plotly.js-basic-dist": "^1.47.3",
+    "plotly.js-basic-dist": "^2.22.0",
```

Comment:
We're already using v2 of plotly.js-basic-dist in synapse-react-client, so this should not cause new problems.